packages/agent/package.json

```diff
@@ -16,8 +16,12 @@
     "test": "mocha -r ts-node/register test/**/*.test.ts",
     "mocha": "mocha -r ts-node/register",
     "start": "parcel serve --out-dir dist/app app/index.html app/harness.ts",
-    "prepack": "tsc --outdir dist --declaration --sourcemap && parcel build --public-url ./ --no-minify --out-dir dist/app app/index.html app/test-frame.html app/harness.ts",
-    "manifest:build": "parcel build test/fixtures/manifest.src.js --out-dir test/fixtures --out-file manifest.js --no-minify --global __bigtestManifest"
+    "prepack:app": "parcel build --public-url ./ --no-minify --out-dir dist/app app/index.html app/test-frame.html app/harness.ts",
+    "prepack:tsc": "tsc --outdir dist --declaration --sourcemap",
+    "prepack": "yarn prepack:app && yarn prepack:tsc",
```

Comment:
There is one danger of making the prepack script use yarn, and that is if someone, using `npm` or `pnpm` or some other package were to want to install this package from a `git` or a `github` url, it would run prepack after installation and potentially blow up. I can't see that as a likely scenario, but wanted to point it out.

Reply:
Is there a better way of invoking one script from another? We don't *have* to do that, but it's convenient/

Reply:
I've never found a good way, and I don't think this should block it. Just pointing it out in case it bites anybody in the future